styles/style.css

```diff
@@ -94,8 +105,25 @@ p {
 
 
 /* Buttons */
+ 
+/* INSERT BUTTON STYLES HERE */
+.donate Btn{
+  background-color:orange;
+  color: white;
+}
+
+.volunteerBtn{
+background-color:white;
+color:orange;
+}
```

Comment:
Your button styling looks fine, however there's 2 things:
1. you don't need to space in `.donate Btn{` - just `.donateBtn`
2. These classes need applying to the appropriate button elements